src/analysis/normed_space/basic.lean

```diff
@@ -1992,3 +1994,67 @@ instance [semi_normed_group V] : normed_group (completion V) :=
 
 end completion
 end uniform_space
+
+namespace locally_constant
+
+variables {X Y : Type*} [topological_space X] (f : locally_constant X Y)
+
+section monoid_hom
+
+variables [monoid Y] [topological_space Y] [has_continuous_mul Y]
+
+/-- The inclusion of locally-constant functions into continuous functions as a multiplicative
+monoid hom. -/
+@[to_additive "The inclusion of locally-constant functions into continuous functions as an
+additive monoid hom."]
+def to_continuous_map_monoid_hom : locally_constant X Y →* C(X, Y) :=
+{ to_fun    := coe,
+  map_one' := by { ext, simp, },
+  map_mul'  := λ x y, by { ext, simp, }, }
+
+@[simp, to_additive] lemma coe_to_continuous_map_monoid_hom :
+  (f.to_continuous_map_monoid_hom : X → Y) = f :=
+rfl
```

Comment:
Does `simps` not generate a better lemma?

Reply:
Oh, in fact it does; thanks.